src/tests/ftest/rebuild/container_create.py

```diff
@@ -0,0 +1,258 @@
+            (when available).  Finally verify that rebuild completes and the
+            pool info indicates the correct number of rebuilt objects and
+            records.  Also confirm that all 1000 additional containers created
+            during rebuild are accessible.
+
+        Use Cases:
+            Basic rebuild of container objects of array values with sufficient
+            numbers of rebuild targets and no available rebuild targets.
+
+        :avocado: tags=all,medium,full_regression,rebuild,rebuildcontcreate
+        """
+        # Get test params
+        targets = self.params.get("targets", "/run/server_config/*")
+        pool_qty = self.params.get("pools", "/run/test/*")
+        loop_qty = self.params.get("loops", "/run/test/*")
+        cont_qty = self.params.get("containers", "/run/test/*")
+        rank = self.params.get("rank", "/run/test/*")
+        node_qty = len(self.hostlist_servers)
+
+        # Get pool params
+        self.pool = []
+        for index in range(pool_qty):
+            self.pool.append(TestPool(self.context, self.log))
+            self.pool[-1].get_params(self)
+
+        # Get ior params
+        ior_cmd = IorCommand()
+        ior_cmd.get_params(self)
+
+        # Cancel any tests with tickets already assigned
+        if rank == 1 or rank == 2:
+            self.cancelForTicket("DAOS-2434")
+
+        errors = [0 for _ in range(loop_qty)]
+        for loop in range(loop_qty):
+            # Log the start of the loop
+            loop_id = "LOOP {}/{}".format(loop + 1, loop_qty)
+            self.log.info("%s", "-" * 80)
+            self.log.info("%s: Starting loop", loop_id)
+
+            # Create the requested number of pools
+            info_checks = []
+            rebuild_checks = []
+            for pool in self.pool:
+                pool.create()
+                info_checks.append(
+                    {
+                        "pi_uuid": pool.uuid,
+                        "pi_ntargets": node_qty * targets,
+                        "pi_nnodes": node_qty,
+                        "pi_ndisabled": 0,
+                    }
+                )
+                rebuild_checks.append(
+                    {
+                        "rs_errno": 0,
+                        "rs_done": 1,
+                        "rs_obj_nr": 0,
+                        "rs_rec_nr": 0,
+                    }
+                )
+
+            # Check the pool info
+            status = True
+            for index, pool in enumerate(self.pool):
+                status &= pool.check_pool_info(**info_checks[index])
+                status &= pool.check_rebuild_status(**rebuild_checks[index])
+            self.assertTrue(
+                status,
+                "Error verifying pool info prior to excluding rank {}".format(
+                    rank))
+
+            # Create a container with 1GB of data in the first pool
+            ior_cmd.flags.update("-v -w -W -G 1 -k", "ior.flags")
```

Comment:
How many objects it will create? just 1?  how does it make sure it will created on the failure rank?